docs/kratos/session-management/10_session-lifespan.mdx

```diff
@@ -9,21 +9,52 @@ sidebar_label: Session lifespan
 Each session is valid for a set amount of time. This time is the session's lifespan. When the session lifespan expires, the user
 must re-authenticate.
 
-In the configuration, the session lifespan is expressed in hours.
-
-Run this command to adjust the session lifespan to 720 hours (30 days):
+In the configuration, the session lifespan is expressed in hours, minutes and seconds combination (hour (h), minute (m), second
+(s) to define interval, for example: **1h1m10s**, **10s**, **1h**).
 
 ````mdx-code-block
 import CodeBlock from '@theme/CodeBlock'
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs>
-  <TabItem value="cloud" label="Ory CLI" default>
+  <TabItem value="console" label="Ory Console" default>
+
+To change the session lifespan, go to [**Ory Console**](https://console.ory.sh/) → **Session Settings**, enter the desired lifespan value (Use hour (h), minute (m), second (s) to define interval, for example: 1h1m10s, 10s, 1h) and click the **Save** button.
+
+For example, to set the session lifespan to 30 days, enter `720h0m0s`.
+
+  </TabItem>
+  <TabItem value="cloud" label="Ory CLI">
+1. Download the Ory Identities config from your project and save it to a file:
+
+```shell
+## List all available projects
+ory list projects
+
+## Get config
+ory get identity-config {project-id} --format yaml > identity-config.yaml
+```
+
+2. Update the configuration value for session lifespan property to the desired value (Use hour (h), minute (m), second (s) to define interval, for example: 1h1m10s, 10s, 1h)
+
+```yaml title="config.yml"
+session:
+  cookie:
+    domain: fancy-moofasa-nsuq4mdx5a.projects.oryapis.com
+    name: ory_session_fancymoofasansuq4mdx5a
+    path: /
+    persistent: false
+    same_site: Lax
+  // highlight-start
+  lifespan: 720h0m0s
```

Comment:
```suggestion
  lifespan: 720h
```
Do you need the minutes and the seconds here?

Reply:
Not necessarily. I just wanted to add that explicitly there so the format is clearer to the reader (by means of repeated reinforcing 😓 )